Library/Formula/pidgin.rb

```diff
@@ -68,7 +68,7 @@ def install
       args << "--without-x"
       args << "--disable-gtkui"
     else
-      args << "--with-x"
+      args << "--without-x"
```

Comment:
You're duping an `args` here. Both the `if build.without?` and `else` now have `args << "--without-x"` lines. Can just stick it in the main block.